README.md

```diff
@@ -125,6 +125,31 @@ Retries are configurable using the following flags
       --retry-min-time duration   Minimum wait interval (default 200ms)
 ```
 
+
+Append group if iam role already exists
+
+```
+aws-auth upsert  --maproles --rolearn arn:aws:iam::00000000000:role/test  --username test   --groups test --append
+```
+
+Append group if iam user already exists
+
+```
+aws-auth upsert  --maproles --rolearn arn:aws:iam::00000000000:role/test  --username test   --groups test --append
+```
+
+Update username during upsert (default is true)
+
+```
+aws-auth upsert  --maproles --rolearn arn:aws:iam::00000000000:role/test  --username test   --groups test
+```
+
+Do not update username during upsert
+
+```
+aws-auth upsert  --maproles --rolearn arn:aws:iam::00000000000:role/test  --username test   --groups test --update-username=false
+```
```

Comment:
Instead of all of these, could we just add two examples?

e.g.

Append groups to mapping instead of overwriting by using `--append`

```
aws-auth upsert --maproles --rolearn arn:aws:iam::00000000000:role/test --username test --groups test --append
```

Avoid overwriting username by using `--update-username=false`

```
aws-auth upsert --maproles --rolearn arn:aws:iam::00000000000:role/test --username test2 --groups test --update-username=false
```